### DISCUSSION TITLE
Listing creation_timestamp Vs original_creation_timestamp

### DISCUSSION THREAD

**Discussion by chrismeats** [etsy/open-api#1012] [Q&A]
Does anyone know the difference between these 2 fields returned on listings?

**Comment by Gareth064** [comment]
I have a slight memory where it was explained like this.

original_creation_timestamp = when you started the draft listing

creation_timestamp = when you first published the listing from draft.

**Comment by chrismeats** [comment]
@Gareth064 Looking at it more I am wondering if it could be based on the last time the listing was renewed?

I am seeing listings that have a fair amount of reviews, view, and favorites, with a creation_timestamp of yesterday but a original creation stamp of just over a year ago.

I'm not sure what else would explain the difference.

**Reply by Gareth064** [reply]
That makes a lot of sense. 

If you want a solid answer then email developer@etsy.com and ask them.

If you get a decent answer then report back here so we can all learn.

**Comment by emergingdzns** [comment]
the original creation timestamp is when the listing was first created. But listings can be renewed over an over again. So the created timestamp is the timestamp that it was last renewed/started.
